src/mono/netcore/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -294,6 +294,11 @@
       <Compile Include="$(BclSourcesRoot)\System\TypeSpec.cs" />
   </ItemGroup>
 
+  <!-- Enable unsupported browser warnings -->
+  <ItemGroup Condition="'$(TargetsBrowser)' == 'true' or '$(TargetsAnyOS)' == 'true'">
```

Comment:
Not sure if we need `'$(TargetsAnyOS)' == 'true'`, it doesn't reveal any new warning though, added it because the project is targeting `<TargetFramework>$(NetCoreAppCurrent)</TargetFramework>`